docs/fuzzing_in_depth.md

```diff
@@ -203,6 +203,9 @@ instances, so running more than one address sanitized target would be a waste.
 *IF* you are running a saturated corpus, then you can run up to half of the 
 instances with sanitizers.
 
+An alternative approach, [SAND](./SAND.md) could combine different sanitizers 
```

Comment:
proposal: "An alternative but more effective approach is to use much [SAND](./SAND.md) which can combine different sanitizers with a much higher throughput."

Reply:
Done